modules/tensor_mechanics/include/materials/NonLocalDamage.h

```diff
@@ -0,0 +1,65 @@
+//* This file is part of the MOOSE framework
```

Comment:
Maybe I'm being picky, but I think it would be more natural to name this NonlocalDamage (since "nonlocal" is a single word). That would be consistent with our naming for things like `NonlinearSystem`.